packages/css/src/plugins/expand-shorthands/utils.ts

```diff
@@ -15,38 +15,48 @@ export const isColor = (node: Node): boolean => {
   return (node.type === 'word' || node.type === 'func') && node.isColor;
 };
 
+const widthUnits = new Set([
```

Comment:
Hey @JakeLane do we know how these units were picked? There may be more units to add https://developer.mozilla.org/en-US/docs/Web/CSS/length

Reply:
no idea, probably good to update - will do 👍

Reply:
Thanks for rearranging them in alphabetical order! :)